www/object_detail.inc

```diff
@@ -239,8 +291,17 @@ function LoadRequests($request_file_name) {
             if (count($columns)) {
                 $request = array();
                 foreach ($columns as $i => $value) {
-                    if (array_key_exists($i + 1, $needed_columns) && strlen(trim($value)))
-                        $request[$needed_columns[$i + 1]] = trim($value);
+                    $trimmed_value = trim($value);
+                    if (array_key_exists($i + 1, $needed_columns) && strlen($trimmed_value)) {
+                        $request_key = $needed_columns[$i + 1];
+                        if (array_key_exists($request_key, $int_fields)) {
+                          $request[$request_key] = (int)$trimmed_value;
+                        } else if (array_key_exists($request_key, $float_fields)) {
+                          $request[$request_key] = (float)$trimmed_value;
+                        } else {
+                          $request[$request_key] = $trimmed_value;
+                        }
+                    }

```

Comment:
Rather than a lookup table I'd rather use a runtime check to see if it is numeric like [here](http://php.net/manual/en/function.is-numeric.php#107326).

Basically turn it into this:
```php
$trimmed_value = trim($value);
if (isset($needed_columns[$i + 1]) && strlen($trimmed_value)) {
  if (is_numeric($trimmed_value)) {
    $request[$needed_columns[$i + 1]] = $trimmed_value + 0;
  } else {
    $request[$needed_columns[$i + 1]] = $trimmed_value;
  }
}
```